go_expvar/README.md

```diff
@@ -1,87 +1,75 @@
-# Go_expvar Integration
+# Go Expvar Integration
 
-## Overview
+# Overview
 
-Get metrics from go_expvar service in real time to:
+Track the memory usage of your Go services and collect metrics instrumented from Go's expvar package.
 
-* Visualize and monitor go_expvar states
-* Be notified about go_expvar failovers and events.
+If you prefer to instrument your Go code with [dogstats-go](https://github.com/DataDog/datadog-go) instead, you can still use this integration to collect memory-related metrics.
 
-## Installation
+# Installation
 
-Install the `dd-check-go_expvar` package manually or with your favorite configuration manager
+The Go Expvar check is packaged with the Agent, so simply [install the Agent](https://app.datadoghq.com/account/settings#agent) anywhere you run Go services whose metrics you want to collect.
```

Comment:
Technically you could have a single agent on a given host configured with multiple go_expvar instances fetching metrics from different urls exposed by different remote machines. This use case would be quite weird and unlikely to happen but still possible.